frontend/demo/component/button/button-labels.ts

```diff
@@ -0,0 +1,82 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+
+import { css, html, LitElement, customElement, internalProperty } from 'lit-element';
+import '@vaadin/vaadin-text-field/vaadin-text-field';
+import '@vaadin/vaadin-button/vaadin-button';
+import '@vaadin/vaadin-ordered-layout/vaadin-vertical-layout';
+import '@vaadin/vaadin-ordered-layout/vaadin-horizontal-layout';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('button-labels')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  static get styles() {
+    return css`
+      vaadin-horizontal-layout {
+        align-items: baseline;
+      }
+    `;
+  }
+
+  @internalProperty()
+  private primaryEmail = 'foo@example.com';
+
+  @internalProperty()
+  private secondaryEmail1 = 'bar@example.com';
+
+  @internalProperty()
+  private secondaryEmail2 = 'baz@example.com';
+
+  render() {
+    return html`
+      <!-- tag::snippet[] -->
+      <vaadin-vertical-layout>
+        <vaadin-horizontal-layout theme="spacing">
+          <vaadin-text-field
+            id="primary-email"
+            label="Primary email address"
+            .value="${this.primaryEmail}"
+            @value-changed="${(e: CustomEvent) => (this.primaryEmail = e.detail.value)}"
+          ></vaadin-text-field>
+          <vaadin-button
+            arial-label="Remove primary email address"
+            @click="${() => (this.primaryEmail = '')}"
+            >Remove</vaadin-button
+          >
+        </vaadin-horizontal-layout>
+        <vaadin-horizontal-layout theme="spacing">
```

Comment:
Separate the compoents for readability. Same applies to all the examples.

```suggestion
        </vaadin-horizontal-layout>
        
        <vaadin-horizontal-layout theme="spacing">
```

Reply:
Added one empty line between layouts and components that span more than 1 line.